CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 - #3555 Content Sync: prevent timeout errors when sync-ing AEM cloud instances with large volumes of data
 - #3560 Redirect Manager: url-encode search terms in Full Text search mode
 - #3562 Fixed compilation errors in iscurrentusermemberof render condition
+- #2745 Fixed circular dependency in EnsureOakIndex
```

Comment:
All items above have been released already in 6.12.0. We need to cleanup changelog before.

Reply:
fixed